### ISSUE TITLE
CatBoostClassifier _classes is None after load_model()

### ISSUE THREAD

**Issue by vcwseas** [catboost/catboost#909]
Problem:

Train a model as cls. Save it. Load it as test.  cls._classes returns labels,  but test. _classes returns None. 

catboost version: 0.15.1
Operating System: MacOS High Sierra
CPU: 2.8 GHz Intel Core i7

**Comment by annaveronika** [comment]
Could you please provide the code that you are running?

**Comment by andrey-khropov** [comment]
This has been fixed in version 0.15.2.

Note that you should use ``classes_``, not ``_classes`` for labels as [specified in CatBoost documentation](https://catboost.ai/docs/concepts/python-reference_catboostclassifier_attributes.html#classes) and [sklearn classifier API](https://scikit-learn.org/stable/developers/contributing.html#specific-models).

Simple example shows that it works now:

```py
#!/usr/bin/env python3

import sys
import numpy as np
import catboost as cb


print ('python version=', sys.version)
print ('numpy version=', np.__version__)
print ('catboost version=', cb.__version__)


features = np.asarray(
    [[0.27290749, 0.63002519, 0., 0.38624339, 0.],
     [0.27290748, 0.63002519, 0., 0.38624339, 0.],
     [0.27290747, 0.63002519, 0., 0.38624339, 0.]]
)
labels = np.asarray([0, 1, 2])

model = cb.CatBoostClassifier(
    iterations=2,
    loss_function='MultiClass'
)

model.fit(features, labels)

print ('classes_ After fitting', model.classes_)

model.save_model('./multiclass_3ints')

loaded_model = cb.CatBoostClassifier()
loaded_model.load_model('./multiclass_3ints')

print ('classes_ Restored from loading', loaded_model.classes_)

```

outputs:

```
python version= 3.6.5 (default, Mar 29 2018, 15:38:43) 
[GCC 4.2.1 Compatible Apple LLVM 8.1.0 (clang-802.0.42)]
numpy version= 1.17.2
catboost version= 0.17.4
0:	learn: 1.0986123	total: 52.6ms	remaining: 52.6ms
1:	learn: 1.0986123	total: 53.1ms	remaining: 0us
classes_ After fitting ['0', '1', '2']
classes_ Restored from loading ['0', '1', '2']
```
